### PR TITLE
Cleaned up MPU interrupt initialisation

### PR DESCRIPTION
**Pull request by blckmn**
Have only tested on the BlueJay - but it is working ok. Really only touched the F4s

### CODE REVIEW

File: src/main/drivers/accgyro_mpu.c [hunk 1]
```diff
@@ -241,254 +223,15 @@ void MPU_DATA_READY_EXTI_Handler(void)
...
-			}
-		}
-		if (timewatchdog == 32) {
-			debug[2]=timewatch[0] << 0 | timewatch[1] << 1 | timewatch[2] << 2  | timewatch[3] << 3  | timewatch[4] << 4  | timewatch[5] << 5  | timewatch[6] << 6  | timewatch[7] << 7 | timewatch[8] << 8 | timewatch[9] << 9 | timewatch[10]<< 10 | timewatch[11]<< 11 | timewatch[12]<< 12 | timewatch[13]<< 13 | timewatch[14]<< 14 | timewatch[15]<< 15;
-			debug[3]=timewatch[16]<< 0 | timewatch[17]<< 1 | timewatch[18]<< 2  | timewatch[19]<< 3  | timewatch[20]<< 4  | timewatch[21]<< 5  | timewatch[22]<< 6  | timewatch[23]<< 7 | timewatch[24]<< 8 | timewatch[25]<< 9 | timewatch[26]<< 10 | timewatch[27]<< 11 | timewatch[28]<< 12 | timewatch[29]<< 13 | timewatch[30]<< 14 | timewatch[31]<< 15;
-			debug[1]++;
-			timewatchdog=0;
-		}
-*/
 #endif
```
Comment (rs2k) [inline src/main/drivers/accgyro_mpu.c:226]: Thanks for getting rid of this. I kept meaning to after I got the data I needed.

File: src/main/sensors/initialisation.c [hunk 2]
```diff
@@ -158,98 +163,6 @@ const extiConfig_t *selectMPUIntExtiConfig(void)
...
-            .gpioPin = Pin_15,
-            .exti_port_source = EXTI_PortSourceGPIOA,
-            .exti_pin_source = EXTI_PinSource15,
-            .exti_line = EXTI_Line15,
-            .exti_irqn = EXTI15_10_IRQn
-    };
-    return &MotolabF3MPU6050Config;
-#endif
-
     return NULL;
```
Comment (rs2k) [inline src/main/sensors/initialisation.c:166]: woohoo! RF's first major bandaid is gone!

Comment (blckmn) [inline src/main/sensors/initialisation.c:166]: I'll look to fix up the other targets and move to one line defines (as per bluejay) for the other targets, and potentially move the F1/3 target exti config over to target.h too.

File: src/main/drivers/accgyro_mpu.c (hunk 1 above)
Comment (blckmn) [inline src/main/drivers/accgyro_mpu.c:226]: I saw it had been commented out for a number of commits - so figured it was good to go.

Comment (rs2k) [inline src/main/drivers/accgyro_mpu.c:226]: Yes, I probably have other code that's just commented out that needs to go. It's a bad habit.

**Comment by blckmn** [comment]
This is good to merge I think - unless you @MJ666 and @rs2k have any feedback?

**Comment by MJ666** [comment]
Looks good to me.

**Pull request merged**